config/ftbquests/quests/chapters/basic_power.snbt

```diff
@@ -0,0 +1,1164 @@
+			rewards: [
+				{
+					id: "16D31D3D55F77046"
+					type: "item"
+					item: "fluxnetworks:flux_dust"
+					count: 2
+					random_bonus: 2
+				}
+				{
+					id: "19CFBA244CF82C53"
+					type: "xp"
+					xp: 10
+				}
+				{
+					id: "5CB51B7DDAE608F1"
+					type: "loot"
+					exclude_from_claim_all: true
+					table_id: 487623848494439020L
+				}
+			]
+		}
+		{
+			title: "The \"Core\" Crafting Materials"
+			x: -9.0d
+			y: 4.0d
+			description: ["In this mod, you'll need &aFlux Cores&r and &aFlux Blocks&r to craft the core parts of your network. Make a few of each!"]
+			dependencies: ["35CC898E0E49FE58"]
+			hide: true
+			id: "1BE26A00A420DAE3"
+			tasks: [
+				{
+					id: "5E760075B279BA71"
+					type: "item"
+					item: "fluxnetworks:flux_core"
+					count: 4L
+				}
+				{
+					id: "01B3CDAA69D1FA2F"
+					type: "item"
+					item: "fluxnetworks:flux_block"
+				}
+			]
+			rewards: [
+				{
+					id: "4FA376B34C2BF04A"
+					type: "item"
+					item: "fluxnetworks:flux_dust"
+					count: 4
+					random_bonus: 4
+				}
+				{
+					id: "3C2BBFC2A36E722F"
+					type: "xp"
+					xp: 100
+				}
+			]
+		}
+		{
+			title: "Starting Your Network"
+			x: -9.0d
+			y: 5.0d
+			description: [
+				"The first item you'll need to start your Flux Network is a &9Flux Plug&r. "
+				""
+				"The Plug is used to \"draw\" power from the block it is attached to. Aside from a small buffer, the Plug does not store power itself, so don't worry about it zapping up all of your power."
+				""
+				"It is suggested to place the Plug on a power storage block like an energy cube. It can connect to cables, pipes, or the output of any power producing machine."
+				""
+				"To learn how to set up your first network, check the next quest!"
+			]
+			dependencies: ["1BE26A00A420DAE3"]
+			id: "5F078A574A783B02"
+			tasks: [{
+				id: "00B1C30B3E10D332"
+				type: "item"
+				item: "fluxnetworks:flux_plug"
+			}]
+			rewards: [
+				{
+					id: "46F6946231FB28A0"
+					type: "item"
+					item: "fluxnetworks:flux_block"
+				}
+				{
+					id: "615F6356856A3371"
+					type: "xp"
+					xp: 100
+				}
+				{
+					id: "116BF6D4838A099B"
+					type: "loot"
+					exclude_from_claim_all: true
+					table_id: 4196188979167302596L
+				}
+			]
+		}
+		{
+			icon: "fluxnetworks:flux_configurator"
+			x: -9.0d
+			y: 6.0d
+			subtitle: "You're a Network Admin Now!"
+			description: [
+				"Right-click on your plug and you'll see the Flux Network UI. On the far top-right corner, click on the + button to create your first network. You'll need to set a password to create the network, but you can also set a color!"
+				""
+				"From here, you can go to the Network Selection tab to activate your network on the plug. If the plug is attached to a power source, you can now harness that power anywhere in your system using a Flux Point!"
+				""
+				"Pro Tip: You can create multiple networks if you want to have different power sources powering different parts of your systems!"
```

Comment:
Another use case for multiple networks: one network to send power from generators into the main battery (usually an Induction Matrix) and another network to power all of the machines and devices that use power.

Furthermore, setting a password isn't necessarily _required_ to create a network; that's just due to the default security setting (Protected), which allows anyone with the correct password to access the network. The setting can be changed to Private so only you can access your networks or to Public to allow anyone on the server to access your network (not recommended on public servers).

Reply:
While the info is good to have, that's a lot for a quest description!